pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -126,6 +126,17 @@ type QueueSortPlugin interface {
 	Less(*PodInfo, *PodInfo) bool
 }
 
+// PostFilterPlugin is an interface for Post-filter plugin. Post-filter is an
+// informational extension point. Plugins will be called with a list of nodes
+// that passed the filtering phase. A plugin may use this data to update internal
+// state or to generate logs/metrics.
+type PostFilterPlugin interface {
+	Plugin
+	// PostFilter is called by the scheduling framework after a list of nodes
+	// passed the filtering phase.
+	PostFilter(pod *v1.Pod, nodes []*v1.Node)
```

Comment:
As @goodluckbot also pointed, `pc *PluginContext` should also be passed. Also, `failedPredicateMap` returned by `findNodesThatFit` should be passed to PostFilter plugins. I think we should also allow them to return errors. To be consistent with other plugins, we can let them return a `*Status`.

Reply:
SGTM

Reply:
@bsalamat I have addressed the comments.  And now, `PostFilter` and `RunPostFilterPlugins` both return a `Status`, but I noticed the following plugins didn't return a `Status` pointer:

```go
	Postbind(pc *PluginContext, p *v1.Pod, nodeName string)
	Unreserve(pc *PluginContext, p *v1.Pod, nodeName string)
```

If it is necessary to return status in these extension points, I could open to PR to solve this.

Reply:
PostBind and Unreserve are informational only and do not need to return anything. PostFilter is informational as well, but it can be used to perform more complex tasks too. That's why I believe we need Status for it.